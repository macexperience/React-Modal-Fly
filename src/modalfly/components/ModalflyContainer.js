import React from 'react';
import './styles.css';

export function ModalflyContainer(props) {
    return (
        <div id='modalfly-container'></div>
    );
}