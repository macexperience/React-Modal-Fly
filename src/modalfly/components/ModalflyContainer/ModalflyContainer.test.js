import React from 'react';
import renderer from 'react-test-renderer';
import { ModalflyContainer } from './ModalflyContainer';

describe('<ModalflyContainer />', () => {
    it('renders and mounts', () => {
        expect.assertions(1);
        const component = renderer.create(<ModalflyContainer />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});