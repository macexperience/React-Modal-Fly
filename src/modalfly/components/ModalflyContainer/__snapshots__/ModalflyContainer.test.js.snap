// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalflyContainer /> renders and mounts 1`] = `
<div
  className="modalflycontainer"
  id="modalfly-container"
  style={
    Object {
      "zIndex": "9999",
    }
  }
/>
`;
